src/utils/blog-helpers.js

```diff
@@ -133,6 +177,21 @@ function generateBlogController(siteID) {
 function generateBlogRouter(siteID) {
   return {
     routes: [
+      {
+        // Get route data
+        method: 'GET',
+        path: `/${siteID}/resolve/:locale/:uuid`,
```

Comment:
so we're going to need to have a UUID for each locale translation this way. Maybe we can agree to a common pattern that we stick to? like adding `-es` / `-zh-Hant` to the end of the en UUID